scripts/new_project.kts

```diff
@@ -189,13 +190,21 @@ object NewProject {
         showMessage("=> 🔎 Renaming package name within files...")
         File(projectPath)
             .walk()
-            .filter { it.name.endsWith(".kt") || it.name.endsWith(".xml") }
+            .filter { it.name.endsWithAny(".kt", ".xml", ".gradle.kts") }
             .forEach { filePath ->
-                rename(
-                    sourcePath = filePath.toString(),
-                    oldValue = TEMPLATE_PACKAGE_NAME,
-                    newValue = packageName
-                )
+                if (filePath.name == "jacoco-report.gradle.kts") {
```

Comment:
Can we use `when(filePath.name)` for this if block? 🤔

Reply:
@lydiasama Updated 💯